README.md

```diff
@@ -250,6 +333,8 @@ The following command runs the unit tests:
 [aws-config]: http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
 [awscli]: https://docs.aws.amazon.com/cli/latest/userguide/installing.html
 [awscli-config]: http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration
+[azure-cli]: https://packages.microsoft.com/yumrepos/azure-cli/azure-cli-2.0.69-1.el7.x86_64.rpm
```

Comment:
```suggestion
[azure-cli]: https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest
```

Reply:
The reason I suggested changing the link is because I think its only used in the powered by section and I thought user facing docs made more sense than an rpm.

Reply:
👍 I concur - user facing docs seem more helpful here.